src/main/java/uk/gov/di/ipv/lambda/CredentialIssuerHandler.java

```diff
@@ -86,7 +80,7 @@ public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent in
     }
 
     private CredentialIssuerConfig getCredentialIssuerConfig(CredentialIssuerRequestDto request) {
-        return CREDENTIAL_ISSUERS.stream()
+        return credentialIssuers.getCredentialIssuerConfigs().stream()
```

Comment:
We need to add some logic to refresh the credential issuers after a certain period, say 5 minutes. I'm thinking this might be a reasonable place to do it. We could have a field on the `CredentialIssuers` object that is a timestamp of when it expires (set when the object is created). We could then check here if it's expired and re-assign `credentialIssuers` with a call to `getCredentialIssuers()` if it has.
We could probably do it as another ticket though. Let's have a chat about it.